config/routes.rb

```diff
@@ -4,7 +4,9 @@
   resources :choices, except: :destroy
   resources :categories, except: :destroy
   resources :communities, shallow: true do
-    resources :indices
+    resources :indices do
+      get :finalize, on: :member
```

Comment:
if this is changing anything about the index this should be a patch operation instead of get